packages/webpack/tsconfig.json

```diff
@@ -2,9 +2,14 @@
   "extends": "../../.config/tsconfig.workspace.json",
   "include": ["src"],
   "compilerOptions": {
+    "lib": ["es2022"],
```

Comment:
we dropped node 16 but it's still in the workspace config for some pkgs that still support it. there's a thread about it somewhere here.

Reply:
I don't want to compromise on JavaScript features because of typescript configuration complexity. I'd rather have idiomatic JavaScript and messy tsconfig than the other way around.